bin/dev.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node_modules/.bin/ts-node
```

Comment:
This was one change we needed to make from the oclif v3 migration guide: https://github.com/oclif/core/blob/main/guides/V3_MIGRATION.md#bin-scripts-for-esmcjs-interoperability

To be clear, all the +/- files in `bin/` are copied straight from oclif generated files.